native_client/ctcdecode/scorer.cpp

```diff
@@ -155,11 +155,35 @@ void Scorer::save_dictionary(const std::string& path, bool append_instead_of_ove
     om = std::ios::out|std::ios::binary;
   }
   std::fstream fout(path, om);
+  if (fout.bad()) {
+    std::cerr << "Error opening '" << path << "'" << std::endl;
+    return;
```

Comment:
This method should be made fallible and the return code should be checked in generate_package.py.

Reply:
Yes, that's what I commented in the issue, but I'm waiting on @carlfm01 feedback of whether this properly catch the error.